### DISCUSSION TITLE
How to synchronize navigation between two panels?

### DISCUSSION THREAD

**Discussion by ronspe** [doublecmd/doublecmd#775] [General]
Hi,

I have been looking for a way to synchronize navigation between two panels but I cannot find it in the documentation and my Google fu gives me no results either. I was hoping somebody can enlighten me.

What I mean is this:
Left panel has open dir: /a/local/folder1/subfolder1 
Right panel har open dir: /a/remote/folder1/subfolder1
So on both panels the folder structure in "folder1" is the same but the paths differ.

What I would like is that when I move on the left panel from "folder1" to a sub-directory, I will do the same in the right directory.
And when I move up a directory I will do the same in the other panel.

I often need to compare files and in combination with the "compare by contents" functionality it would guarantee a very smooth workflow.
Is this possible? 

Thanks!

**Comment by alexx2000** [comment]
Hi, yes. See [cm_SyncChangeDir](https://doublecmd.github.io/doc/en/cmds.html#cm_SyncChangeDir) internal command, you can attach a hotkey to it or add it to the toolbar.

**Reply by ronspe** [reply]
Hi Alex,

Thank you!

Cheers, Ronald.
